rebar.config

```diff
@@ -1,17 +1,36 @@
-{deps, []}.
+{cover_export_enabled, true}.
+
+{cover_enabled, true}.
+
+{deps, [
+    {erlang_lorawan, {git, "https://github.com/helium/erlang-lorawan.git"}},
+    {grpc_client, {git, "https://github.com/Bluehouse-Technology/grpc_client.git"}},
+    {grpcbox, {git, "https://github.com/novalabsxyz/grpcbox.git"}},
+    {helium_proto, {git, "https://github.com/helium/proto.git"}},
+    {lager, "3.9.2"}
+]}.
 
 {erl_opts, [
     debug_info,
+    {parse_transform, lager_transform},
     warnings_as_errors
 ]}.
 
+{grpc, [
+    {protos, "protos"},
+    {gpb_opts, [{module_name_suffix, "_pb"}]}
+]}.
+
 {plugins, [
+    {grpcbox_plugin, {git, "https://github.com/novalabsxyz/grpcbox_plugin.git"}},
     {rebar3_format, "1.2.1"},
     {erlfmt, "1.0.0"}
 ]}.
 
+{pre_hooks, []}.
```

Comment:
as per previous comment, better to call the grpc makefile target here from a compile pre hook.  likewise for a clean prehook.  see https://github.com/helium/miner/blob/master/rebar.config#L61

Reply:
(as above: let's merge with the current balance of Makefile vs rebar.config and revisit in a future PR if needed)